app/models/searchgov_url.rb

```diff
@@ -33,7 +33,9 @@ class SearchgovUrl < ActiveRecord::Base
   counter_culture :searchgov_domain, column_name: 'urls_count'
   counter_culture :searchgov_domain,
     column_name: proc {|url| !url.fetched? ? 'unfetched_urls_count' : nil },
-    column_names: { ['searchgov_urls.last_crawl_status is null'] => 'unfetched_urls_count' }
+    column_names: { ['searchgov_urls.last_crawled_at IS NULL'] => 'unfetched_urls_count' }
```

Comment:
This tweak is to ensure that `unfetched` here means the same thing as it does in `Fetchable`'s `unfetched` scope: 
```ruby
scope :unfetched, -> { where('ISNULL(last_crawled_at)') }
```